src/main/java/UI/GameBoard.java

```diff
@@ -35,17 +35,144 @@ private class GameBoardPanel extends JPanel {
         protected void paintComponent(Graphics g) {
             super.paintComponent(g);
 
-            // Calculate the starting point to make the square centered
-            int x = (getWidth() - 600) / 2;
-            int y = (getHeight() - 600) / 2;
+            // Draw the title "Monopoly" at the top middle
+            drawGameTitle(g);
 
-            // Draw the square
-            g.drawRect(x, y, 600, 600);
+            // Draw the overall Monopoly board layout
+            drawBoard(g);
         }
+
+        private void drawGameTitle(Graphics g) {
+            // Set the title font, style and size
+            Font titleFont = new Font("Arial", Font.BOLD, 30);
+            g.setFont(titleFont);
+
+            // Get the width and height of the string with the current font to position it correctly
+            FontMetrics fm = g.getFontMetrics();
+            int titleWidth = fm.stringWidth("Monopoly: Adventure at UofT");
+            int titleHeight = fm.getAscent();  // This gives the height from base to top of the font
+
+            // Calculate the position to center the title
+            int x = (getWidth() - titleWidth) / 2;
+            int y = titleHeight + 20;  // A little padding from the top
+
+            // Set color for the title
+            g.setColor(new Color(173, 216, 230));  // Light blue color
```

Comment:
It's better to put specific colors that you plan to use on the top of the file, so it allows for easier change later on.